src/main/java/org/broadinstitute/hellbender/utils/hdf5/HDF5PoN.java

```diff
@@ -16,6 +16,10 @@
 /**
  * HDF5 File backed Panel of Normals data structure.
  *
+ * Several attributes are stored transposed, in order to dodge a very slow write time in HDF5.
```

Comment:
Be more explicit about the slow write time, i.e. say (as I assume is the reason) that you want a smaller number of rows, hence need samples to be rows and targets to be columns.